src/puja_poreddiwar/Assignmt78.java

```diff
@@ -0,0 +1,46 @@
+package puja_poreddiwar;
+
+public class Assignmt78 {
+
+	void printPattern1() {
+		int num = 4;
+		for (int i = 1; i <= num; i++) {
```

Comment:
Make code generic so if you want to print for 10 or 2 rows then also it is possible